Dockerfile

```diff
@@ -0,0 +1,72 @@
+# golang is based on debian:jessie
```

Comment:
Can you describe what is this dockerfile for? Basically, provide a documentation for a person that sees it for the first time in their life :)